ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -113,6 +117,8 @@ fun WearApp(watchRepository: WatchRepository) {
             currentBackStackEntry?.arguments?.getSerializable(SCROLL_TYPE_NAV_ARGUMENT)
                 ?: DestinationScrollType.NONE
 
+        var displayValue by remember { mutableStateOf(2) }
```

Comment:
This should be a more descriptive name... what is it a display value for?

Reply:
I updated the variable names to be more descriptive and pushed the updates.  For example, the 'ValueDisplay' route name is now called 'UserInputComponents', etc